drivers/ethernet/dsa_ksz8xxx.c

```diff
@@ -884,9 +938,9 @@ struct net_pkt *dsa_ksz8xxx_xmit_pkt(struct net_if *iface, struct net_pkt *pkt)
 		port_idx = (1 << (ctx->dsa_port_idx));
 	}
 
-	NET_DBG("TT - port: 0x%x[%p] LEN: %d 0x%x 0x%x 0x%x 0x%x 0x%x 0x%x",
-		port_idx, iface, len, lladst.addr[0], lladst.addr[1],
-		lladst.addr[2], lladst.addr[3], lladst.addr[4], lladst.addr[5]);
+	NET_DBG("TT - port: 0x%x[%p] LEN: %d 0x%x 0x%x 0x%x 0x%x 0x%x 0x%x", port_idx, iface, len,
```

Comment:
Please change all `NET_DBG` calls to `LOG_DBG` in this file. The former is only meant for core net stack internal use.